### ISSUE TITLE
ObjectFinder.find() crashed JVM

### ISSUE THREAD

**Issue by ccyydt** [bytedeco/javacv#210]
hi,guys,try the code behind.
```java
    public static void main(String args[]) {
        for (int j = 0; j < 500; j++) {
            System.out.println("b~~~~~~~~~" + j);
            BufferedImage buff = new BufferedImage(1920, 1080, IPL_DEPTH_8U);  //if i comment this linde only, as j increased, maybe 100+ ,200+ or 300+,codes crashed.
            opencv_core.IplImage srcImage = cvLoadImage("d:/wts/src.png", CV_LOAD_IMAGE_GRAYSCALE);  //crash happen at this line.
            opencv_core.IplImage tarImage = cvLoadImage("d:/wts/tar.png", CV_LOAD_IMAGE_GRAYSCALE);  //image which is a part of srcImage 
            ObjectFinder.Settings settings = new ObjectFinder.Settings();
            settings.setUseFLANN(true);
            settings.setRansacReprojThreshold(5);
            settings.setObjectImage(tarImage);
            ObjectFinder finder = new ObjectFinder(settings);
            double[] result = null;
            finder.find(srcImage);  //if i comment this line only,the codes not crashed
            if (result != null) {
                System.out.println("find object");
            }
            cvReleaseImage(tarImage);
            cvReleaseImage(srcImage);
        }  
    }
```
//if i don't comment any line,codes crashed when second loop.
when crashed, i got message like these:
```
A fatal error has been detected by the Java Runtime Environment:
EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x6d7e2460, pid=6680, tid=5592.....
```
I think probelm is inside of  ObjectFinder.find(),maybe there is some memery leak or some other problem.
I used javacv 0.7 , jre1.7 ,opencv2.4.8, and I also try  javacv 1.0 , jre1.8 ,opencv2.4.8. the codes always  crashed. My system is win7 64bit.
You can run these codes,see what happens.Maybe you can give some suggestion. thanks.

**Comment by saudet** [comment]
I think I've fixed this issue in the commit above. Please let me know if this solves the crash on your side as well, and thanks for reporting!

**Comment by saudet** [comment]
Fix included in version 1.1. Thanks for reporting!
